src/cmd/mod.rs

```diff
@@ -87,6 +89,7 @@ pub enum ApiErrorKind {
     NoApiRoute {},
     NoLongerSupported {},
     NoMatchingExit {},
+    NotFound {},
```

Comment:
I would prefer to continue using specific errors like we do in other cases. So `AppleTransactionNotFound` or something? Then any UI matching on the error code doesn't need to worry about the source. (Or we add a field for the type of thing not found?)

Thoughts @FlorianUekermann?

Reply:
Yeah, if this should really never happen `BadRequest` can be used as a generic client error indicator. The text description can point to the actual problem.